modules/java_api/build.gradle

```
@@ -14,7 +14,7 @@ def nativesCPP;
 def openvinoVersion = "2024.2"
 
 def native_resources = []
-def tbb_dir = System.getenv('TBB_DIR')
+def tbb_dir = System.getenv('TBB_DIR') ?: "/openvino-android/one-tbb-install/lib/cmake/TBB"
```

Shouldn't we define the full path to `openvino-android` repo? Please see: https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/build_android.md#create-a-work-directory

Something like:
```suggestion
def tbb_dir = System.getenv('TBB_DIR') ?: System.getenv('OPV_HOME_DIR') + "/one-tbb-install/lib/cmake/TBB"
```